layers/auto-completion/README.org

```diff
@@ -118,8 +118,8 @@ The variable has no effect when =auto-complete= is used.
 
 #+BEGIN_SRC emacs-lisp
 (setq-default dotspacemacs-configuration-layers
-  '(auto-completion :variables
-                    auto-completion-enable-sort-by-usage t))
+  '((auto-completion :variables
+                    auto-completion-enable-sort-by-usage t)))
 #+END_SRC
```

Comment:
@robbyoconnor No, the new one is correct. You need parenthesis around the layer when there are `:variables`.